aunt-leahs-app/src/components/header.jsx

```diff
@@ -1,20 +1,19 @@
 import React from 'react';
-import SubText from './SubText.jsx';
-import constants from '../../constants.jsx';
-import '../../styles.css';
+import constants from '../constants.jsx';
+import '../styles.css';
 
 const headerText = constants.HEADER.MAIN_HEADER;
 
 const Head = ({ page }) => {
     /*takes in 'page' as props. Refer to ./components/constants.jsx for props.page values */
     return(
         <div className='headerBox'>
-                <img className="headerImage header"  alt="Aunt Leah's Logo" src={require('../../AuntLeahsTrees.png')}/>
+                <img className="headerImage header"  alt="Aunt Leah's Logo" src={require('../AuntLeahsTrees.png')}/>
                 <div className="headerMain header">{headerText.headTwo}</div>
                 {(page !== null) 
```

Comment:
This looks wrong (but isn't) because it would seem like you'd normally want to render the whole sub-header even if there was no String for it.
It took me a while to realize that it is your intent to not render any sub-header in certain scenarios (i.e. for admin pages).
I think that that would be more clear if you make a const at the start and then use that const for the check. For example:
```
const hasSubHeader = (page !== null);
...
hasSubHeader &&
    <div className...
```

Reply:
noted